capstone-ui/src/components/Match/Match.jsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import "./Match.css";
+import Loading from "../Loading/Loading";
+import { FaHeart, FaRegHeart } from "react-icons/fa";
+
+export default function Match({
```

Comment:
Good job here. 
This component presents a good opportunity to modularizing a little bit more. We can have a <Matches/> containing multiple <MatchCard/>s.